packages/widgetbook/lib/src/navigation/nodes/widgetbook_use_case.dart

```diff
@@ -21,29 +22,36 @@ class WidgetbookUseCase extends WidgetbookNode {
   factory WidgetbookUseCase.center({
     required String name,
     required Widget child,
+    String? designLink,
```

Comment:
We do not need to update this API, since it's deprecated, and no new features should be introduced to it.

Reply:
Attended.